daemon/cmd/daemon_main.go

```diff
@@ -800,9 +800,19 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Int(option.ToFQDNsMinTTL, defaults.ToFQDNsMinTTL, "The minimum time, in seconds, to use DNS data for toFQDNs policies")
 	option.BindEnv(vp, option.ToFQDNsMinTTL)
 
+	flags.Bool(option.EnableStandaloneDNSProxy, false, "Enables standalone dns proxy.")
+	option.BindEnv(vp, option.EnableStandaloneDNSProxy)
+
+	flags.Bool(option.EnableEmbeddedDNSProxy, true, "Enables embedded DNS proxy.")
+	flags.MarkHidden(option.EnableEmbeddedDNSProxy)
+	option.BindEnv(vp, option.EnableEmbeddedDNSProxy)
+
 	flags.Int(option.ToFQDNsProxyPort, 0, "Global port on which the in-agent DNS proxy should listen. Default 0 is a OS-assigned port.")
 	option.BindEnv(vp, option.ToFQDNsProxyPort)
 
+	flags.Int(option.ToFqdnsServerPort, 40045, "Global port on which the grpc server for standalone dns proxy should listen. Default is 40045.")
```

Comment:
nit: no need to mention the default, this will be automatically added to the generated usage based on the second argument. Also, let's properly capitalize abbreviations while at it:

```suggestion
	flags.Int(option.ToFqdnsServerPort, 40045, "Global port on which the gRPC server for standalone DNS proxy should listen")
```

Reply:
Updated it !